docs/advanced-topics/snapshot-and-doi.rst

```diff
@@ -0,0 +1,101 @@
+Creating Snapshots and Requesting DOIs
+=============================================
+
+Introduction
+------------------
+Starting in Dockstore 1.7.0, users can create snapshots and Digital
+Object Identifiers (DOIs) for their workflows. The ‘Snapshot’ and
+‘Request DOI’ actions can be found on the versions tab of any published
+workflow on the ‘My Workflows’ page. These features are specific to
+individual versions of a workflow entry.
+
+.. image:: ../assets/images/docs/snapshot_doi/snapshot_doi_versiontab.png
+
+
+
+Connect Zenodo Account
+----------------------
+Dockstore uses `Zenodo <http://about.zenodo.org/>`__ as its DOI provider. A Zenodo account is
+required to request a DOI for a workflow version. Link your Zenodo credentials to your Dockstore account on the
+`accounts page <https://dockstore.org/accounts?tab=accounts>`__. Requesting a DOI on Dockstore will
+create a public entry on Zenodo and upload associated files.
```

Comment:
Can also note (if not present later), a side effect of using Zenodo credentials is a user would be able to go in and fine tune things on Zenodo